File: src/main/java/it/ipzs/qeeaissuer/dto/Cnf.java
package it.ipzs.qeeaissuer.dto;

import java.io.Serializable;

public class Cnf implements Serializable {
	private static final long serialVersionUID = 4211723201991928614L;

	private JWKDto jwk;

	public JWKDto getJwk() {
		return jwk;
	}

	public void setJwk(JWKDto jwk) {
		this.jwk = jwk;
	}
}

File: sdk-android/androidpidprovider/src/main/java/it/ipzs/androidpidprovider/entity/Cnf.kt
package it.ipzs.androidpidprovider.entity

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

internal class Cnf : Serializable {

    @SerializedName("jwk")
    @Expose
    val jwk: Jwk? = null

}